packages/discord.js/src/util/Options.js

```diff
@@ -67,6 +67,17 @@ const { version } = require('../../package.json');
  * @returns {IShardingStrategy} The strategy to use for sharding
  */
 
+/**
+ * A function to change the concurrency handling for shard identifies of this manager
+ * ```js
+ * const concurrency = 1;
+ * (manager) => new SimpleIdentifyThrottler(concurrency);
```

Comment:
Please adjust the example to reflect the default used in `@discordjs/ws` and not a hardcoded concurrency of 1.
https://github.com/discordjs/discord.js/blob/d26e022afcc84448bb91deb4dd621aea20df6617/packages/ws/src/utils/constants.ts#L32-L35